jaxb-ri/external/rngom/pom.xml

```diff
@@ -77,6 +77,52 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-compile</id>
+                        <configuration>
+                            <source>9</source>
+                            <target>9</target>
+                            <includes>
+                                <include>module-info.java</include>
+                            </includes>
+                            <compilerArgs>
+                                <arg>--add-reads</arg>
+                                <arg>com.sun.tools.rngom=relaxngDatatype</arg>
+                            </compilerArgs>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>base-compile</id>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <configuration>
+                            <source>1.7</source>
+                            <target>1.7</target>
+                            <excludes>
+                                <exclude>module-info.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.0.0-M1</version>
```

Comment:
For some unknown to me reason, when using 3.0.0 ant is put on CP instead of module path. XJC/JXC will than have --add-reads XJC=ALL-UNNAMED in order to generate javadoc. More specific --add-reads XJC=ant from automatic module path looks better to me. Left as 3.0.0-M1 for now.